### ISSUE TITLE
`console` error in Obsidian 1.3.x: a `Component` should be passed to `renderMarkdown`

### ISSUE THREAD

**Issue by dljsjr** [jamiebrynes7/obsidian-todoist-plugin#234]
**Describe the bug**
I'm not actually sure that this is a bug yet, but Obsidian itself seems to feel that this could be the source of a potential memory leak.

When the plugin parses a todoist fence in the latest version of Obsidian (1.3.4 as I'm writing this), the following stacktrace is emitted:

```
Error: Plugin "todoist-sync-plugin" is not passing Component in renderMarkdown. This is needed to avoid memory leaks when embedded contents register global event handlers.
    at t.render (app.js:1:1255232)
    at t.renderMarkdown (app.js:1:1255013)
    at eval (MarkdownRenderer.svelte:13:28)
    at run (index.mjs:20:12)
    at Array.map (<anonymous>)
    at eval (index.mjs:2058:58)
    at flush (index.mjs:1280:17)
t.render @ app.js:1
```

**To Reproduce**
1. Install the latest Obsidian
2. Install this plugin
3. Observe the developer console while opening a page with a <pre>```todoist</pre> fence

**Expected behavior**
No errors in the console.

**Screenshots**
![image](https://github.com/jamiebrynes7/obsidian-todoist-plugin/assets/444635/33545357-146f-4c49-b0f6-cd16d64d8424)

**Desktop (please complete the following information):**
 - Plugin Version: **_1.11.1_**
 - Obsidian Version: **_1.3.4_**

**Additional context**
N/A

**Comment by seagullmouse** [comment]
Yep this resolved itself for me.

**Comment by algaspar** [comment]
I am getting this same error.  I don't see any actual errors on the pages where I am showing todoist tasks, but I see this in the developer console.  Should I just ignore this?

Thanks--

Al

**Comment by seatrout** [comment]
I also am getting this error on the console. It appears with a couple of other plugins, too (custom-classes and rss-reader). I am not a real coder at all, but so far as I can see, you are passing "null" into renderMarkdown where it is expecting the name of the plugin. I will make this change and see if (1) it eliminates the error and (2) the plugin still works!

**Comment by ybalkind** [comment]
> I also am getting this error on the console. It appears with a couple of other plugins, too (custom-classes and rss-reader). I am not a real coder at all, but so far as I can see, you are passing "null" into renderMarkdown where it is expecting the name of the plugin. I will make this change and see if (1) it eliminates the error and (2) the plugin still works!

@seatrout - I'm experiencing exactly that same as what you described (other plugins affects, the "null" issue). I know its almost a year ago, but did you resolve? What is "this change" you are referring too (if you still remember)?

**Comment by jamiebrynes7** [comment]
I'm actually completely rewriting the rendered task lists which will fix this as well. So this will be resolved in the next release

**Comment by jamiebrynes7** [comment]
This should be fixed as of the latest release - v2.0.0
